performance/pom.xml

```diff
@@ -34,6 +34,7 @@
         <beanvalidation-impl.version>none</beanvalidation-impl.version>
 
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
+        <automatic.module.name>org.hibernate.validator.performance</automatic.module.name>
```

Comment:
I don't think this one is necessary: we don't publish anything and it's not supposed to be used.